### ISSUE TITLE
Call to undefined scope when using `whereDate`

### ISSUE THREAD

**Issue by brendt** [nunomaduro/larastan#90]
I've got this snippet:

```php
UnitPrice::query()
    ->whereUnit($unitPrice->unit)
    ->whereKeyNot($unitPrice->id)
    ->inFuture()
    ->get()
    ->each
    ->delete();
```

This is the `inFuture` scope definition on the model:

```php
public function scopeInFuture(Builder $builder): Builder
{
    return $builder->whereDate('date_from', '>', now()->toDateString());
}
```

I'm getting this error:

```php
Call to an undefined method Illuminate\Database\Eloquent\Builder::inFuture().
```

It seems like #28 fixed some scoped issues, and I can't help but feel this one is related to #89, as the error only occurs for scopes using `whereDate`.

**Comment by nunomaduro** [comment]
@brendt This issue got addressed on the version `v0.2.11`. Can you test it?

**Comment by brendt** [comment]
@nunomaduro Seems to be fixed, thanks!
